diff --git a/node_modules/@emotion/native/dist/native.browser.esm.js b/node_modules/@emotion/native/dist/native.browser.esm.js
index 706e626..ace3ba7 100644
--- a/node_modules/@emotion/native/dist/native.browser.esm.js
+++ b/node_modules/@emotion/native/dist/native.browser.esm.js
@@ -1,5 +1,6 @@
-import reactNative, { StyleSheet } from 'react-native';
-import { createStyled, createCss } from '@emotion/primitives-core';
+import * as reactNative from "react-native";
+import { StyleSheet } from "react-native";
+import { createStyled, createCss } from "@emotion/primitives-core";
 
 /**
  * a function that returns a styled component which render styles in React Native
@@ -8,8 +9,57 @@ import { createStyled, createCss } from '@emotion/primitives-core';
 var styled = createStyled(StyleSheet);
 
 var css = createCss(reactNative.StyleSheet);
-var components = ['ActivityIndicator', 'ActivityIndicatorIOS', 'ART', 'Button', 'DatePickerIOS', 'DrawerLayoutAndroid', 'Image', 'ImageBackground', 'ImageEditor', 'ImageStore', 'KeyboardAvoidingView', 'ListView', 'MapView', 'Modal', 'NavigatorIOS', 'Picker', 'PickerIOS', 'ProgressBarAndroid', 'ProgressViewIOS', 'ScrollView', 'SegmentedControlIOS', 'Slider', 'SliderIOS', 'SnapshotViewIOS', 'Switch', 'RecyclerViewBackedScrollView', 'RefreshControl', 'SafeAreaView', 'StatusBar', 'SwipeableListView', 'SwitchAndroid', 'SwitchIOS', 'TabBarIOS', 'Text', 'TextInput', 'ToastAndroid', 'ToolbarAndroid', 'Touchable', 'TouchableHighlight', 'TouchableNativeFeedback', 'TouchableOpacity', 'TouchableWithoutFeedback', 'View', 'ViewPagerAndroid', 'WebView', 'FlatList', 'SectionList', 'VirtualizedList'];
-var index = components.reduce(function (acc, comp) {
+var components = [
+  "ActivityIndicator",
+  "ActivityIndicatorIOS",
+  "ART",
+  "Button",
+  "DatePickerIOS",
+  "DrawerLayoutAndroid",
+  "Image",
+  "ImageBackground",
+  "ImageEditor",
+  "ImageStore",
+  "KeyboardAvoidingView",
+  "ListView",
+  "MapView",
+  "Modal",
+  "NavigatorIOS",
+  "Picker",
+  "PickerIOS",
+  "ProgressBarAndroid",
+  "ProgressViewIOS",
+  "ScrollView",
+  "SegmentedControlIOS",
+  "Slider",
+  "SliderIOS",
+  "SnapshotViewIOS",
+  "Switch",
+  "RecyclerViewBackedScrollView",
+  "RefreshControl",
+  "SafeAreaView",
+  "StatusBar",
+  "SwipeableListView",
+  "SwitchAndroid",
+  "SwitchIOS",
+  "TabBarIOS",
+  "Text",
+  "TextInput",
+  "ToastAndroid",
+  "ToolbarAndroid",
+  "Touchable",
+  "TouchableHighlight",
+  "TouchableNativeFeedback",
+  "TouchableOpacity",
+  "TouchableWithoutFeedback",
+  "View",
+  "ViewPagerAndroid",
+  "WebView",
+  "FlatList",
+  "SectionList",
+  "VirtualizedList"
+];
+var index = components.reduce(function(acc, comp) {
   return Object.defineProperty(acc, comp, {
     enumerable: true,
     configurable: false,
diff --git a/node_modules/@emotion/native/dist/native.esm.js b/node_modules/@emotion/native/dist/native.esm.js
index 706e626..ace3ba7 100644
--- a/node_modules/@emotion/native/dist/native.esm.js
+++ b/node_modules/@emotion/native/dist/native.esm.js
@@ -1,5 +1,6 @@
-import reactNative, { StyleSheet } from 'react-native';
-import { createStyled, createCss } from '@emotion/primitives-core';
+import * as reactNative from "react-native";
+import { StyleSheet } from "react-native";
+import { createStyled, createCss } from "@emotion/primitives-core";
 
 /**
  * a function that returns a styled component which render styles in React Native
@@ -8,8 +9,57 @@ import { createStyled, createCss } from '@emotion/primitives-core';
 var styled = createStyled(StyleSheet);
 
 var css = createCss(reactNative.StyleSheet);
-var components = ['ActivityIndicator', 'ActivityIndicatorIOS', 'ART', 'Button', 'DatePickerIOS', 'DrawerLayoutAndroid', 'Image', 'ImageBackground', 'ImageEditor', 'ImageStore', 'KeyboardAvoidingView', 'ListView', 'MapView', 'Modal', 'NavigatorIOS', 'Picker', 'PickerIOS', 'ProgressBarAndroid', 'ProgressViewIOS', 'ScrollView', 'SegmentedControlIOS', 'Slider', 'SliderIOS', 'SnapshotViewIOS', 'Switch', 'RecyclerViewBackedScrollView', 'RefreshControl', 'SafeAreaView', 'StatusBar', 'SwipeableListView', 'SwitchAndroid', 'SwitchIOS', 'TabBarIOS', 'Text', 'TextInput', 'ToastAndroid', 'ToolbarAndroid', 'Touchable', 'TouchableHighlight', 'TouchableNativeFeedback', 'TouchableOpacity', 'TouchableWithoutFeedback', 'View', 'ViewPagerAndroid', 'WebView', 'FlatList', 'SectionList', 'VirtualizedList'];
-var index = components.reduce(function (acc, comp) {
+var components = [
+  "ActivityIndicator",
+  "ActivityIndicatorIOS",
+  "ART",
+  "Button",
+  "DatePickerIOS",
+  "DrawerLayoutAndroid",
+  "Image",
+  "ImageBackground",
+  "ImageEditor",
+  "ImageStore",
+  "KeyboardAvoidingView",
+  "ListView",
+  "MapView",
+  "Modal",
+  "NavigatorIOS",
+  "Picker",
+  "PickerIOS",
+  "ProgressBarAndroid",
+  "ProgressViewIOS",
+  "ScrollView",
+  "SegmentedControlIOS",
+  "Slider",
+  "SliderIOS",
+  "SnapshotViewIOS",
+  "Switch",
+  "RecyclerViewBackedScrollView",
+  "RefreshControl",
+  "SafeAreaView",
+  "StatusBar",
+  "SwipeableListView",
+  "SwitchAndroid",
+  "SwitchIOS",
+  "TabBarIOS",
+  "Text",
+  "TextInput",
+  "ToastAndroid",
+  "ToolbarAndroid",
+  "Touchable",
+  "TouchableHighlight",
+  "TouchableNativeFeedback",
+  "TouchableOpacity",
+  "TouchableWithoutFeedback",
+  "View",
+  "ViewPagerAndroid",
+  "WebView",
+  "FlatList",
+  "SectionList",
+  "VirtualizedList"
+];
+var index = components.reduce(function(acc, comp) {
   return Object.defineProperty(acc, comp, {
     enumerable: true,
     configurable: false,
